src/js/renderer.js

```diff
@@ -31,8 +31,11 @@ define(["stage", "worker!layout-worker.js"], function(stage, worker) {
         pending = source;
       }
     },
-    getImage: function() {
-      var svgXml = stage.svg();
+    getImage: function(obj) {
```

Comment:
This method defaults export options to `zoom: 0` but `stage.svg()` accepts `reset: true`.